src/common/components/Radio/Radio.tsx

```diff
@@ -1,20 +1,55 @@
-import * as React from 'react'
+import React, { useEffect, useState } from 'react'
+import cn from 'classnames'
 
-import * as Styles from './Radio.styles'
-
-interface RadioProps {
+export type TRadioProps = {
   isChecked: boolean
-  clickHandler: (event: React.MouseEvent<HTMLLabelElement, MouseEvent>) => void
+  clickHandler: (param: boolean) => void
+  className?: string
+  smallCircleClass?: string
   children: React.ReactNode
 }
 
-function Radio({ children, isChecked, clickHandler }: RadioProps): JSX.Element {
+const Radio = ({
+  children,
+  isChecked,
+  clickHandler,
+  className = 'w-5 h-5',
+  smallCircleClass = 'w-2 h-2',
+}: TRadioProps): JSX.Element => {
+  const [checked, setChecked] = useState<boolean>(isChecked)
```

Comment:
We do we need both isChecked from props and local checked?
I guess having only isChecking and clickHandler from props is enough

Reply:
fixed